h/views/accounts.py

```diff
@@ -323,13 +332,13 @@ def get_when_not_logged_in(self):
                     _(
                         "We didn't recognize that activation link. "
                         "Have you already activated your account? "
-                        'If so, try <a href="{url}">logging in</a> using the username '
+                        "If so, try logging in using the username "
                         "and password that you provided."
-                    ).format(url=self.request.route_url("login"))
+                    ),
```

Comment:
This flash message is now shown on the login page so it doesn't need to link to the login page